dist/pagination.js

```diff
@@ -313,6 +316,20 @@
           html += formattedString;
         }
 
+        if(isMultPageSize){
+          self.model.currPageSize = $('.paginationjs-page-combobox').val();
```

Comment:
With global selector you can't use more than one pagination element on page.
You can do smth like this
```javascript
self.model.currPageSize = self.model.el.find('.paginationjs-page-combobox').val();
```